ykrt/src/compile/jitc_yk/jit_ir/jit_ir.y

```diff
@@ -62,22 +62,22 @@ Inst -> Result<ASTInst, Box<dyn Error>>:
     "*" Operand "=" Operand { Ok(ASTInst::Store{tgt: $2?, val: $4?, volatile: false}) }
   | "*" Operand "=" Operand "," "VOLATILE" { Ok(ASTInst::Store{tgt: $2?, val: $4?, volatile: true}) }
   | "BLACK_BOX" Operand { Ok(ASTInst::BlackBox($2?)) }
-  | "GUARD" "TRUE" "," Operand {
-      Ok(ASTInst::Guard{operand: $4?, is_true: true})
+  | "GUARD" "TRUE" "," Operand "," "[" LocalsList "]" {
```

Comment:
We possibly want this to be `OperandsList` later so we can track constants too. I believe constants can end up here via constant propagation.